makara.gemspec

```diff
@@ -7,6 +7,10 @@ Gem::Specification.new do |gem|
   gem.description   = %q{Read-write split your DB yo}
   gem.summary       = %q{Read-write split your DB yo}
   gem.homepage      = ""
+  gem.licenses      = ['MIT']
+  gem.metadata      = {
+                        source_code_uri: 'https://github.com/taskrabbit/makara'
```

Comment:
Note that RubyGems requires metadata keys and values to be strings. The Symbol key causes gemspec validation errors that Bundler exposes.

Reply:
Looks like this has been reported & fixed in #244.